Makefile

```diff
@@ -14,7 +14,7 @@ COPYRIGHT_VALUE ?= $(shell grep -i ^copyright LICENSE | sed 's/^[Cc]opyright //'
 
 OS := $(shell uname | tr '[:upper:]' '[:lower:]')
 ARCH := $(shell uname -m | if grep -q x86_64 ; then echo amd64 ; else uname -m ; fi)
-GOPATH := $(shell echo "$${GOPATH%%:*}")
+GOPATH := $(shell go env GOPATH)
```

Comment:
:heart: in my browser history top 10: http://www.tldp.org/LDP/abs/html/string-manipulation.html